assemblies/assembly_openshift-freeform/main.adoc

```diff
@@ -25,6 +26,16 @@ ifdef::PAM[]
 ** {Central} Monitoring requires a 64Mi PV.
 ** Each {KIE_SERVER} deployment by default requires one 1Gi PV for the database. You can change the database PV size in the template parameters. If you deploy multiple separate {KIE_SERVERS}, each of the servers requires a separate database PV. This requirement does not apply if you use an external database server.
 * Your OpenShift environment supports persistent volumes with `ReadWriteMany` mode. For information about access mode support in OpenShift Online volume plug-ins, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/architecture/#pv-access-modes[Access Modes].
++
+[IMPORTANT]
```

Comment:
@mramendi I didn't take the time to look at every rendered output, just a couple, but I did notice in just those few that this ReadWriteMany not is aligned with the bullet here but not in the other outputs. I think aligning with the bullet is correct, and I suggest updating all other instances to be the same way.